sotodlib/preprocess/preprocess_plot.py

```diff
@@ -219,8 +222,11 @@ def plot_sso_footprint(aman, planet_aman, sso, wafer_offsets=None, filename='./s
                          'ws5': (8.5, 5),
                          'ws6': (8.5, -7)}
 
-    # Get default focal plane from sotodlib
-    hw = np.load('/so/home/msilvafe/shared_files/sat_hw_positions.npz')
+    if focal_plane is None:
+        # Get default focal plane from sotodlib
+        hw = np.load('/so/home/msilvafe/shared_files/sat_hw_positions.npz')
```

Comment:
@skhrg does this exist somewhere office that we can point to by default? It's really just wafer outlines in xi-eta...we could even probably define this in a tiny file that lives in sotodlib potentially.